sparkles/__init__.py

```diff
@@ -1,4 +1,19 @@
-__version__ = '4.1.2'
+from pkg_resources import get_distribution
+
+try:
+    _dist = get_distribution(__package__)
+    __version__ = _dist.version
+
+    if not __file__.startswith(_dist.location):
```

Comment:
Both values `__file__` and `_dist.location` need to be `.lower()` because Windows can return different cases.

Reply:
If I understand correctly, these are the only lines required to install these packages in the simplest cases. The question is whether we want to set the __version__ variable for use within python anyway.